app/schemas/email.py

```
@@ -0,0 +1,53 @@
+
+
+class EmailRequest(BaseModel):
+    """Request model for sending an email"""
+    from_address: EmailStr = Field(..., alias="from", description="Header From (RFC 5322.From)")
+    envelope_from: Optional[EmailStr] = Field(None, description="Envelope Sender (RFC 5321.MailFrom)")
+    to: list[EmailStr] = Field(..., min_length=1, description="Recipient addresses")
+    cc: Optional[list[EmailStr]] = Field(None, description="CC addresses")
+    bcc: Optional[list[EmailStr]] = Field(None, description="BCC addresses")
+    subject: str = Field(..., description="Email subject")
+    body: str = Field(..., description="Email body (plain text or HTML)")
+    html: Optional[bool] = Field(False, description="Whether body is HTML")
```

Ruff (0.14.10)

Boolean positional value in function call

(FBT003)